src/components/shared/Entity/Actions/TestButton.tsx

```diff
@@ -51,7 +51,12 @@ function EntityTestButton({
                         id={`${messagePrefix}.test.waitMessage`}
                     />
                 }
-                actionComponent={<LogDialogActions close={closeLogs} />}
+                actionComponent={
+                    <LogDialogActions
+                        close={closeLogs}
+                        closeCtaKey="cta.dismiss"
+                    />
```

Comment:
Passing in the key to make it dismiss only for test since it feels like the save dialog should still show close.